spec/examples.txt

```diff
@@ -1,21 +0,0 @@
-example_id                                      | status  | run_time        |
```

Comment:
I think, you should add this file to `.gitignore`

Reply:
![image](https://user-images.githubusercontent.com/47780452/203841119-c5a3119c-12ae-47bd-bff8-22b6e2d153fc.png)

Reply:
i have it